src/workspaces/dashboard/WorkspaceDashboard.ts

```diff
@@ -106,6 +106,9 @@ export const WorkspaceDashboard = forwardRef(
           [h(WorkspaceNotifications, { workspace })]
         ),
       ]),
+      // "dummy" div to make background color below the right box sections the same if they do not
+      // fill the full height of the dashboard
+      div({ style: { ...Style.dashboard.rightBox, gridRow: '2', gridColumn: '2' } }, []),
```

Comment:
Without this, you see a white background under the RHS blocks if they are all collapsed and the workspace description isn't long enough to force the first row's height to fill the viewport.

<img width="764" alt="image" src="https://github.com/user-attachments/assets/b5527c25-e02a-4d82-be79-3f1de785efc9">
<img width="920" alt="image" src="https://github.com/user-attachments/assets/3faa7121-6604-4b8d-8b9b-4cff0629121c">